### DISCUSSION TITLE
Using RSPress As Module Tools Document

### DISCUSSION THREAD

**Discussion by WinmezzZ** [web-infra-dev/modern.js#4958] [Q&A]
I found a plugin named `@modern-js/plugin-rspress` , but no more description about it, is this the way use rspress in `modern.js` ?

**Comment by 10Derozan** [comment]
In the past, we have been trying to use this solution internally and getting more optimizations and improvements. In the next version, I will add usage methods on the official website and support in script: `modern new`
By the way, this plugin is only used in module tools. Are you using it?
